### PR TITLE
Moved the metadata into setup.cfg

### PR DESCRIPTION
**Pull request by KOLANICH**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,2 +1,36 @@
+[metadata]
+name = pathspec
+version = 0.5.9
```
Comment (cpburnz) [inline setup.cfg]: The version is outdated. It should be "0.8.0.dev1".

File: setup.cfg [hunk 2]
```diff
@@ -1,2 +1,36 @@
...
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: Implementation :: CPython
+    Programming Language :: Python :: Implementation :: PyPy
+    Topic :: Software Development :: Libraries :: Python Modules
+    Topic :: Utilities
+
+[options]
+packages = find:
+python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*
+setup_requires = setuptools>=44; wheel; setuptools_scm[toml]>=3.4.3
```
Comment (cpburnz) [inline setup.cfg]: *wheel* shouldn't be a hard setup requirement because I only use that when publishing.

File: setup.cfg (hunk 1 above)
Comment (KOLANICH) [reply setup.cfg]: O shi, this shouldn't have been here. The version is meant to be set via git tags.

File: setup.cfg (hunk 2 above)
Comment (KOLANICH) [reply setup.cfg]: In fact pip builds wheels, if you have pip, you have wheel (though inlined into pip).

**Comment by cpburnz** [comment]
I'll accept this pull request if you limit it to just pulling the metadata from `__init__.py` into `setup.cfg`.

`pyproject.toml` and its interaction with *setuptools* is thoroughly lacking in documentation so I'm not interested in adding that until it's properly supported. I don't want `version.py` magically generated from a random commit.

**Comment by KOLANICH** [comment]
As you wish.

**Comment by KOLANICH** [comment]
@cpburnz

**Pull request merged**